hackrsource/MOC/newsapp/views.py

```diff
@@ -35,11 +35,24 @@ def bookmark(request):
 	articleID = int(articleId)
 	user = User.objects.get(id = 1) #will be made dynamic once users login is added
 	article = tbl_MST_NewsArticle.objects.get(articleId = articleID)
-	newBookmark = tbl_TRN_NewsBookmark(userId = user, articleId=article, bookmarkedAt = datetime.now())
+	newBookmark = tbl_TRN_NewsBookmark(user = user, article = article, bookmarkedAt = datetime.now())
 	newBookmark.save()
-	print newBookmark
 	i = newBookmark.bookmarkId
-	return HttpResponse("Success with bookmark id = " + str(i))
+	return HttpResponse("Success bookmark id = " + str(i))
+
+def unbookmark(request):
+	articleId = request.GET.get('id', None)
+	articleID = int(articleId)
```

Comment:
you can just override the variable, no need to create multiple variables with multiple naming conventions,